AstSemantics.md

```diff
@@ -249,11 +249,10 @@ All control flow structures, except `case`, are statements.
  * `if_else`: if statement with then and else bodies
  * `br`: branch to a given label in an enclosing construct (see below)
  * `br_if`: conditionally branch to a given label in an enclosing construct
- * `tableswitch`: a jump table which may jump either to enclosed `case` blocks
+ * `tableswitch`: a jump table which may jump either to enclosed `case` nodes
```

Comment:
"jump to either an immediate child case or a label..."

To make it clear that one can't jump into the middle of a further nested tableswitch.